templates/vehicle/kia.yaml

```diff
@@ -0,0 +1,19 @@
+template: kia
+description: Kia
+params:
```

Comment:
Könnten wir die "dummen" Parameter nicht mit irgendeinem Includemechanismus, z.B. Golang Templates, hier rein bauen? Das ist in fast jedem Auto x-fach wiederholt und wenn die Identifiers x-fach dazu kommen gibts genau so oft auch Anpassungen.

Dopplengen vermeiden!

Reply:
Was sind für dich denn "dumme" Parameter? Meinst du `title`, `user`, `password`, `vin`, `capacity`? Also die am häufigsten auftreten und dann 1 Mal definiert werden. Und für die Ausnahmen (z.b. evnotify) dann halt nicht nehmen.

Reply:
Genau, da kann identifiers gleich mit rein

Reply:
da Du dann 2x templates parsen musst würde es sich anbieten, andere Klammerung zu nehmen (bei go template einstellbar), einmal für "prepare" und einmal für "render". Sonst kommen die sich ins Gehege

Reply:
Über templates wurde es echt komplex. Habe jetzt `paramsbase:` hinzugefügt, da ist momentan nur `vehicles` definiert. In `Params` kann man die dann ergänzen oder von bestehenden `Default` oder `Example` anpassen.